drivers/gles3/storage/particles_storage.cpp

```diff
@@ -271,13 +271,13 @@ void ParticlesStorage::particles_set_fractional_delta(RID p_particles, bool p_en
 
 void ParticlesStorage::particles_set_trails(RID p_particles, bool p_enable, double p_length) {
 	if (p_enable) {
-		WARN_PRINT_ONCE("The OpenGL 3 renderer does not support particle trails");
+		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle trails");
```

Comment:
Stylistic change for consistency with other messages:

```suggestion
		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle trails.");
```